main.go

```diff
@@ -77,9 +76,14 @@ func mainE(ctx context.Context) error {
 
 	flag.IntVar(&logLevel, "v", 0, "Number for the log level verbosity")
 
+	opts := zap.Options{
+		Development: false,
+	}
+	opts.BindFlags(flag.CommandLine)
+
 	flag.Parse()
 
-	ctrl.SetLogger(zap.New(zap.Level(zapcore.Level(-logLevel))))
+	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
```

Comment:
the linter didn't like that line (neither did i) so we read the loglevel directly from flags into options instead